### PR TITLE
Clarify TRTEP doc

### PR DESCRIPTION
**Pull request by yf711**
### Description
* Clarify TRTEP version info

### Motivation and Context
https://github.com/microsoft/onnxruntime/issues/15118: issue to request
https://github.com/microsoft/onnxruntime/pull/16465: previous PR to update trtep doc

### CODE REVIEW

File: docs/build/eps.md [hunk 1]
```diff
@@ -95,19 +95,21 @@ See more information on the TensorRT Execution Provider [here](../execution-prov
 {: .no_toc }
 
 * Install [CUDA](https://developer.nvidia.com/cuda-toolkit) and [cuDNN](https://developer.nvidia.com/cudnn)
-   * The TensorRT execution provider for ONNX Runtime is built and tested up to CUDA 11.8 and cuDNN 8.9.
+   * The TensorRT execution provider for ONNX Runtime 1.15 is built and tested up to CUDA 11.8 and cuDNN 8.9.
```
Comment (jywu-msft) [inline docs/build/eps.md:98]: do we need to add 1.15 here? The original intention is to say that the latest version of OnnxRuntime has been tested with those versions.
what happens if we stay with cuda 11.8 for 1.15, then would we list both 1.15 and 1.16 then? i'd prefer to leave the version information in the requirements table. in the TensorRT-ExecutionProvider.md

File: docs/build/eps.md [hunk 2]
```diff
@@ -95,19 +95,21 @@ See more information on the TensorRT Execution Provider [here](../execution-prov
...
    * The path to the CUDA installation must be provided via the CUDA_PATH environment variable, or the `--cuda_home` parameter. The CUDA path should contain `bin`, `include` and `lib` directories.
    * The path to the CUDA `bin` directory must be added to the PATH environment variable so that `nvcc` is found.
    * The path to the cuDNN installation (path to cudnn bin/include/lib) must be provided via the cuDNN_PATH environment variable, or `--cudnn_home` parameter.
      * On Windows, cuDNN requires [zlibwapi.dll](https://docs.nvidia.com/deeplearning/cudnn/install-guide/index.html#install-zlib-windows). Feel free to place this dll under `path_to_cudnn/bin`  
  * Install [TensorRT](https://developer.nvidia.com/tensorrt)
-   * The TensorRT execution provider for ONNX Runtime is built and tested up to TensorRT 8.6.
+   * The TensorRT execution provider for ONNX Runtime 1.15 is built and tested up to TensorRT 8.6.
```
Comment (diablodale) [inline docs/build/eps.md:104]: This "1.15" is the same inquiry of @jywu-msft above 🤔

File: docs/build/eps.md (hunk 1 above)
Comment (yf711) [reply docs/build/eps.md]: I see. The `1.15` has been removed, and at the end of this line I added a shortcut to the requirement table for more version info

File: docs/build/eps.md (hunk 2 above)
Comment (yf711) [reply docs/build/eps.md]: Thanks for the review! Just updated

**Review by jywu-msft — APPROVED** [review]

**Pull request merged**